### ISSUE TITLE
Expand definition of links within an RC

### ISSUE THREAD

**Issue by neutrinog** [unfoldingWord-dev/Resource-Container#38]
@jag3773 I'd like to propose allowing relative file paths as a valid RC link. For example:
 
This is what we currently support for internal links.
* `[[other/nation]]`

Add this
* `[[../other/nation.md]]`

Doing so would allow links to work online when viewed on gogs. So in the end we'll support both. Where the form we currently support will simply be the shorthand version.

EDIT:
--
I was looking at this again and in order for github or gogs to display the links correct they'd need to be formatted like the following:

```
[Nation](../other/nation/01.md)
```
Which is a little more involved and less readable. 

A different/another change?
--
This leads me to thinking about how we are storing things in containers. We have been strict in saying that things must be broken up by chapter and chunk, or if you don't want everything to be chunked you can just keep it all in a `01` chunk, which to be honest has always been slightly annoying to me and not very clear.

What if we instead allow chapters to exist as a single file like they already are e.g. `abomination.md` rather than `abomination/01.md`?
* the RC links already support this
* it is more readable
* it is easier to maintain the content

In summary chapters could be chunked `abomination/01.md...02.md...03.md etc` or not `abomination.md`.

**Comment by neutrinog** [comment]
In addition to the above changes we may also want to define relative links as "not having a preceding forward slash" while links to another RC "do have a preceding forward slash". This will eliminate some ambiguity.

**Comment by jag3773** [comment]
That's all fine with me.
